### PR TITLE
Rewrite disaster recovery based on recreate and status check

### PR DESCRIPTION
**Pull request by AnnaSjerling**
This PR rewrites the disaster recovery docs based on the new recreate and status check procedures. 

With regards to the functional changes there are three major things to note. 
1. The recovery of the system database is the same as before, even though the way we check if the system db is write available or not has changed. 
2. The step which removes lost servers now also includes recreating some databases. It was decided this mixing of recovering servers and databases is necessary to get recreate into the guide in the best way we could see. 
3. Previously it was not clear how to handle quarantined databases, even though the guide mentioned them in the intro. Now, it is more explicitly discussed how they are supposed to be handles during disaster recovery.

In the past, we have also gotten feedback that the disaster recovery docs are hard to follow. Therefore, this PR also includes refactoring which introduces a new guide structure. This new structure provides more explanations of why we are asking the user to do a certain thing.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Review by tselmegbaasan — COMMENTED** [review]
Just a few comments from testing this out locally by hand.

**Review by tselmegbaasan — APPROVED** [review]
Looks good. I just have a few questions and suggestions.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 1]
```diff
@@ -5,157 +5,292 @@
...
+
+[source, shell]
+----
+CALL dbms.cluster.statusCheck(["system"]);
+----
+
 [NOTE]
+=====
+The write availability of a database configured to have a single primary cannot be checked with the status check, instead check that the primary is allocated on an available server and that it has `currentStatus` = `STARTED`.
+The procedure will still produce an accurate result if all but one primary have been lost during a disaster.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:102]: I wanted to explain that even though it does not work for a standalone, it will still work for a clustered database which has only one allocation left. But that might be too much, I'll just remove it.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 2]
```diff
@@ -5,157 +5,292 @@
...
+This section of the disaster recovery guide uses `neo4j-admin`, for more information about the used commands, see xref:tools/neo4j-admin/index.adoc#neo4j-admin-commands[neo4j-admin commands].
+=====
+
+. Shut down the Neo4j process on all servers.
+This causes downtime for all databases in the cluster until the processes are started again at the end of this section.
+. On each server, run `bin/neo4j-admin dbms unbind-system-db` to reset the `system` database state on the servers.
+. On each server, run `bin/neo4j-admin database info system` and compare the `lastCommittedTransaction` to find out which server has the most up-to-date copy of the `system` database.
+. On the most up-to-date server, run `bin/neo4j-admin database dump system --to-path=[path-to-dump]` to take a dump of the current `system` database and store it in an accessible location.
+. For every _lost_ server, add a new *unconstrained* one according to xref:clustering/servers.adoc#cluster-add-server[Add a server to the cluster].
+It is important that the new servers are unconstrained, or deallocating servers in the next step of this guide might be blocked, even though enough servers were added.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:125]: I don't think that adds very much without an explanation on how to go about doing that. And since I don't think that belong here I'll leave it as is.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 3]
```diff
@@ -5,157 +5,292 @@
...
 +
-or
+[NOTE]
+=====
+While recommended, it is not strictly necessary to add new servers in this step.
+However, not adding new servers reduces the capacity of the cluster to handle work.
+Furthermore, it might require the topology for a database to be altered to make deallocating servers and recreating databases possible.
+=====
+
+. For each stopped database (`currentStatus`= `offline`), start them by running `START DATABASE stopped-db`.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:184]: We said that since we recommend using the status check procedure to find out which databases are write unavailable, we would also recommend all stopped databases being started so that it gives the correct result. The next sentence describes the moving part "This is necessary since stopped databases cannot be moved from one server to another." I will change moved from one server to another. -> deallocated from a server. And also add a comment on how this interacts with status check.

[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:184]: I agree that this is good context to give, however, I don't think it belongs in this section since it would then describe the overall goal in the middle of the steps to reach that goal. I have added some sentences to the beginning of the steps to try and explain this. There is a comment there as well, so I'll try to clarify the motivation there.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 4]
```diff
@@ -5,157 +5,292 @@
...
+
+[source, cypher]
+----
+SHOW SERVERS;
+----
+
+==== Path to correct state
+The following steps can be used to remove lost servers and add new ones to the cluster.
+That includes moving any potential database allocations from lost servers to available servers.
+These steps might also recreate some databases, since a database which has lost a majority of its primary allocations cannot be moved from one server to another.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:166]: Right, a database can still have a quorum left, but have some quarantined allocations. I'll clarify this and the overall goal of the steps.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 5]
```diff
@@ -5,157 +5,292 @@
...
+
+The `system` database contains the view of the cluster.
+This includes which servers and databases are present, where they live and how they are configured.
+During a disaster, the view of the cluster might need to change to reflect a new reality, for example by removing lost servers.
+Databases might also need to be recreated to regain write availability.
+Because both of these steps are executed by modifying the `system` database, making the `system` database write available is a vital first step during disaster recovery.
+
+==== Example verification
+The `system` database's write availability can be verified by using the xref:clustering/monitoring/status-check.adoc#monitoring-replication[Status check] procedure.
+The procedure should be called on all remaining primary allocations of the `system` database, in order to provide the correct view.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:89]: Good point, I'll leave it out here and a more throughout explanation can be in the status check docs.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 6]
```diff
@@ -5,157 +5,292 @@
...
-+
-[NOTE]
+==== Path to correct state
+Start the Neo4j process on all servers that are _offline_.
+If a server is unable to start, inspect the logs and contact support personnel.
+The server may have to be considered indefinitely lost.
+
+[[restore-the-system-database]]
+=== `System` database write availability
+
```
Comment (Frannie-Ludmilla) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:75]: This section is empty. Don't know if it's intentional because we're listing sth :/

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/5/clustering/disaster-recovery)

**Review by NataliaIvakina — COMMENTED** [review]
@AnnaSjerling hey! I went through your guide. Wow! Tremendous work! I left some editorial comments. My idea was to tie up the beginning with the rest of the guide.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 7]
```diff
@@ -5,157 +5,279 @@
...
+[source, cypher]
+----
+SHOW SERVERS;
+----
 
-. Run `SHOW SERVERS`.
-If *all* servers show health `AVAILABLE` and status `ENABLED` continue to xref:clustering/disaster-recovery.adoc#recover-databases[Recover databases].
+==== Path to correct state
+The following steps can be used to remove lost servers and add new ones to the cluster.
+To be able to remove lost servers, any allocations it should host needs to be moved to available servers in the cluster.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:160]:
```suggestion
To be able to remove lost servers, any allocations it should host need to be moved to available servers in the cluster.
```

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 8]
```diff
@@ -5,157 +5,279 @@
...
+This is done in two steps, first any databases that cannot move by themselves needs to be recreated so that they are forced to move.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:161]:
```suggestion
This is done in two steps:

. Any databases that cannot move by themselves must be recreated so that they are forced to move.
```

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 9]
```diff
@@ -5,157 +5,279 @@
...
+Then, any allocations that can move will be told to do so by deallocating the server.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:162]:
```suggestion
. Any allocations that can move will be instructed to do so by deallocating the server.
```

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 10]
```diff
@@ -5,157 +5,279 @@
...
+====
 
 There are three main steps to recovering a cluster from a disaster.
-Completing each step, regardless of the disaster scenario, is recommended to ensure the cluster is fully operational.
-
-. Ensure the `system` database is available in the cluster.
-The `system` database defines the configuration for the other databases; therefore, it is vital to ensure it is available before doing anything else.
+First, ensure the `system` database is write available.
+Then, detach any potential lost servers from the cluster and replace them by new ones.
+Finish disaster recovery by starting or continuing to manage databases and verify that they are write available.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: I added the list, but I don't think we should link to that part of the guide here. First of all, I think it promotes only jumping there and not considering the previous steps. Furthermore, since the others don't have similar links it makes it asymmetrical. I might add the links to all sections thought. What do you think?

[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: Now when I look at it, I'm wondering if it is strange to have it in list format and not mention the first step. Should I add the "Neo4j process started" section also in this overview?

[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: Actually, I had the same question 🙂. But then I decided it was done on purpose. I mean, here you describe three steps of disaster recovery specifically. You cannot do anything without the Neo4j process started. This makes no sense to do anything without starting Neo4j. That's why this step is beyond the described steps of disaster recovery. That was my understanding.

[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: I think it is not important for understanding the overall flow, but what this step wants to cover is if you have a disaster where some process might have been stopped without the user meaning to.

[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: In that case we don't want them to e.g. get back that the system database is write unavailable just because they haven't checked this.

[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: hmm, in this case there four steps:
1. Start the neo4j process on all stopped servers
2. make the 'system db write available
3. check the servers availability
4. make the databases write available again
Correct?

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 11]
```diff
@@ -5,157 +5,279 @@
...
-If *all* servers in a Neo4j cluster are lost in a data center failover, it is not possible to recover the current cluster.
-You have to create a new cluster and restore the databases.
-See xref:clustering/setup/deploy.adoc[Deploy a basic cluster] and xref:clustering/databases.adoc#cluster-seed[Seed a database] for more information.
+If *all* servers in a Neo4j cluster are lost in a disaster, it is not possible to recover the current cluster.
+You have to create a new cluster and restore the databases, see xref:clustering/setup/deploy.adoc[Deploy a basic cluster] and xref:clustering/databases.adoc#cluster-seed[Seed a database] for more information.
 ====
 
 == Faults in clusters
 
 Databases in clusters follow an allocation strategy.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:21]: "Databases in clusters may be allocated differently within the cluster and may also have different numbers of primaries and secondaries."

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 12]
```diff
@@ -5,157 +5,279 @@
...
 Losing a server in a cluster may cause some databases to lose a member while others are unaffected.
-Consequently, in a disaster where multiple servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
+Therefore, in a disaster where one or more servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
 
-== Guide to disaster recovery
+== Guide structure
+[NOTE]
+====
+In this guide, an _offline_ server is a server that is not running but may be restartable.
+A _lost_ server, however, is a server that is currently not running and cannot be restarted.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:31]: Capital letter

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 13]
```diff
@@ -5,157 +5,279 @@
...
+A _write available_ database is able to serve writes, while a _write unavailable_ database is not.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:32]: Capital letter. What is the reason for not making capital letters here?

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 14]
```diff
@@ -5,157 +5,279 @@
...
+First, ensure the `system` database is write available.
+Then, detach any potential lost servers from the cluster and replace them by new ones.
+Finish disaster recovery by starting or continuing to manage databases and verify that they are write available.
 
-. After the `system` database's availability is verified, whether recovered or unaffected by the disaster, recover the lost servers to ensure the cluster's topology meets the requirements.
+Every step consists of the following three sections:
 
-. After the `system` database is available and the cluster's topology is satisfied, you can manage the databases.
+. A state that the cluster needs to be in, with optional motivation.
+. An example of how the state can be verified.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:43]: Does this make it seem too much like "this is the way to do it" rather than the softer suggestion?

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 8 above)
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:161]: databases -> allocations

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 15]
```diff
@@ -5,157 +5,279 @@
...
+
+[CAUTION]
+====
+Remember, recreating a database takes an unbounded amount of time since it may involve copying the store to a new server, as described in xref:clustering/databases.adoc#recreate-databases[Recreate databases].
+Therefore, an allocation with `currentStatus` = `STARTING` will probably reach the `requestedStatus` given some time.
+====
+
+[[example-verification]]
+==== Example verification
+All databases' write availability can be verified by using the xref:clustering/monitoring/status-check.adoc#monitoring-replication[Status check] procedure.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:243]: all clustered databases'

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 16]
```diff
@@ -5,157 +5,279 @@
...
-+
-Add new servers and enable them and then return to step 3, see xref:clustering/servers.adoc#cluster-add-server[Add a server to the cluster] for more information.
-. Run `SHOW SERVERS YIELD *` once all enabled servers host the requested databases (`hosting`-field contains exactly the databases in the `requestedHosting` field), and proceed to the next step.
-Note that this may take a few minutes.
-. For each deallocated server, run `DROP SERVER deallocated-server-id`.
-. Return to step 1.
 
 [[recover-databases]]
-=== Recover databases
+=== Database availability
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:224]: I do not agree with inspect in this sections. If you have done some of the other steps before, this might mean you only need to inspect thing here. However, if no servers were lost this is actually your main step if recreating databases.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 10 above)
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: Yes, maybe we should just add that in, even though the first step is very short. That is also how I presented it for field. I'll do that!

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 17]
```diff
@@ -5,157 +5,279 @@
...
 Databases in clusters follow an allocation strategy.
 This means that they are allocated differently within the cluster and may also have different numbers of primaries and secondaries.
-The consequence of this is that all servers are different in which databases they are hosting.
+The consequence of this is that all servers may be different in which databases they are hosting.
 Losing a server in a cluster may cause some databases to lose a member while others are unaffected.
-Consequently, in a disaster where multiple servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
+Therefore, in a disaster where one or more servers go down, some databases may keep running with little to no impact, while others may lose all their allocated resources.
 
-== Guide to disaster recovery
+== Guide structure
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:27]: "Overview of  the guide"? To make it more in line with other titles.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 18]
```diff
@@ -5,157 +5,279 @@
...
-. *Restore the `system` database.*
-+
-[NOTE]
+==== Path to correct state
+Start the Neo4j process on all servers that are _offline_.
+If a server is unable to start, inspect the logs and contact support personnel.
+The server may have to be considered indefinitely lost.
+
+[[restore-the-system-database]]
+=== `System` database write availability
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:74]: Lets discuss this one together with all other titles.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 16 above)
[the remark this answers is not in the document]
Comment (NataliaIvakina) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:224]: This makes sense. I have not thought about this phase from your point of view, that it could be the main step in recovering the cluster.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 19]
```diff
@@ -5,157 +5,279 @@
...
 
-. If you have previously dropped databases as part of this guide, re-create each one from a backup.
-See the xref:database-administration/standard-databases/create-databases.adoc[Create databases] section for more information on how to create a database.
-. Run `SHOW DATABASES`.
-If all databases are in desired states on all servers (`requestedStatus`=`currentStatus`), disaster recovery is complete.
-// . For each database that remains unavailable, refer to <<unavailable-databases, Managing unavailable databases in a cluster>>.
-// Perform the actions required to get the database available then return to step 2.
+[NOTE]
+=====
+The write availability of a database configured to have a single primary cannot be checked with the status check, instead check that the primary is allocated on an available server and that it has `currentStatus` = `STARTED`.
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:252]: Added "by running `SHOW DATABASES`." Also changed the same text in 2 more places so they match.

**Comment by AnnaSjerling** [comment]
I have printed the wrong words when describing output from e.g. `SHOW DATABASES`. So I fixed the words and the capitalisation of them to match the actual output.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 10 above)
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:38]: With the links it will be more clear that this is an overview.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 13 above)
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:32]: Yes capital letters

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 14 above)
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:43]: Verifying the state

File: modules/ROOT/pages/clustering/disaster-recovery.adoc (hunk 16 above)
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:224]: Make databases write available

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 20]
```diff
@@ -5,157 +5,279 @@
...
 ====
-+
+
+The `system` database contains the view of the cluster.
+This includes which servers and databases are present, where they live and how they are configured.
+During a disaster, the view of the cluster might need to change to reflect a new reality, for example by removing lost servers.
+Databases might also need to be recreated to regain write availability.
+Because both of these steps are executed by modifying the `system` database, making the `system` database write available is a vital first step during disaster recovery.
+
+==== Example verification
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:87]: Verifying the state

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 21]
```diff
@@ -5,157 +5,279 @@
...
+====
+All servers in the cluster's view are available and enabled.
+====
+
+A lost server will still be in the `system` database's view of the cluster, but in an unavailable state.
+Furthermore, according to the view of the cluster, these lost servers are still hosting the databases they had before they became lost.
+Therefore, informing the cluster of servers which are lost is not enough.
+The databases hosted on lost servers also need to be moved onto available servers in the cluster, before the lost servers can be removed.
+
+==== Example verification
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:149]: Verifying the state

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 22]
```diff
@@ -5,157 +5,279 @@
...
+If they are still desired to be in stopped state, run `STOP DATABASE started-db WAIT`.
+
+[CAUTION]
+====
+Remember, recreating a database takes an unbounded amount of time since it may involve copying the store to a new server, as described in xref:clustering/databases.adoc#recreate-databases[Recreate databases].
+Therefore, an allocation with `currentStatus` = `STARTING` will probably reach the `requestedStatus` given some time.
+====
+
+[[example-verification]]
+==== Example verification
```
[the remark this answers is not in the document]
Comment (AnnaSjerling) [reply modules/ROOT/pages/clustering/disaster-recovery.adoc:242]: Verifying the state

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/2025.01/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/2025.01/clustering/disaster-recovery)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1890.surge.sh

Updated pages:
- [Managing databases in a cluster](https://neo4j-docs-operations-1890.surge.sh/operations-manual/2025.01/clustering/databases)
- [Disaster recovery](https://neo4j-docs-operations-1890.surge.sh/operations-manual/2025.01/clustering/disaster-recovery)

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 23]
```diff
@@ -69,30 +69,33 @@ See xref:clustering/setup/routing.adoc#clustering-routing[Server-side routing] f
...
-The `system` database is write available.
+The `system` database is able to accept write operations.
 ====
 
 The `system` database contains the view of the cluster.
 This includes which servers and databases are present, where they live and how they are configured.
 During a disaster, the view of the cluster might need to change to reflect a new reality, such as removing lost servers.
 Databases might also need to be recreated to regain write availability.
-Because both of these steps are executed by modifying the `system` database, making the `system` database write available is a vital first step during disaster recovery.
+Because both of these steps are executed by modifying the `system` database, making the `system` database write-enabled is a vital first step during disaster recovery.
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:95]: This is in your hands, but I don't think it nice to change from a technically correct wording which we have defined in the beginning of the document to multiple adjacent ones which are not defined. Like write-enabled, able to accept write operations and write-capable.

File: modules/ROOT/pages/clustering/disaster-recovery.adoc [hunk 24]
```diff
@@ -167,9 +171,9 @@ SHOW SERVERS;
 ----
 
 ==== Path to correct state
-The following steps can be used to remove lost servers and add new ones to the cluster.
-To be able to remove lost servers, any allocations it should host need to be moved to available servers in the cluster.
-This is done in two different ways:
+Use the following steps to remove lost servers and add new ones to the cluster.
+To remove lost servers, any allocations they were hosting must be moved to available servers in the cluster.
+This can be done in two different ways:
```
Comment (AnnaSjerling) [inline modules/ROOT/pages/clustering/disaster-recovery.adoc:176]: This is not correct, it is not an either or, but both will be needed in most cases.
